src/common/example/index.html

```diff
@@ -1,20 +1,13 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<title>SlidePane Example</title>
+	<title>Example</title>
 	<meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1">
```

Comment:
Just being slightly nitpicky, but everything else looks so good! Could we do `<meta name="viewport" content="width=device-width, initial-scale=1">` and drop the min/max scale? They tend to be bad for usability, especially for people who zoom in manually to read.